os/arch/arm/src/stm32l4/stm32l4_idle.c

```diff
@@ -67,11 +75,72 @@
 #endif
 
 #define PM_IDLE_DOMAIN 0 /* Revisit */
+#define STM32L4_EXT_JOYKEY_ACTIVITY			4
+
+#ifdef CONFIG_SCHED_TICKSUPPRESS
+#ifndef CONFIG_STM32L4_RTC
+#error "Cannot support ticksuppress without RTC enabled (Enable CONFIG_STM32L4_RTC)"
+#endif
+#endif
 
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+#ifdef CONFIG_ARCH_SUPPORT_PM_UTIL
+int pm_sleep(void)
```

Comment:
Because this is in arch and it's chip-specific function, let's rename as up_pmsleep.

Reply:
ok, will update